packages/core/admin/admin/src/translations/en.json

```diff
@@ -626,6 +626,13 @@
   "components.Blocks.modifiers.underline": "Underline",
   "components.Blocks.modifiers.strikethrough": "Strikethrough",
   "components.Blocks.modifiers.code": "Code",
+  "components.Blocks.blockItems.text": "Text",
+  "components.Blocks.blockItems.heading1": "Heading 1",
+  "components.Blocks.blockItems.heading2": "Heading 2",
+  "components.Blocks.blockItems.heading3": "Heading 3",
+  "components.Blocks.blockItems.heading4": "Heading 4",
+  "components.Blocks.blockItems.heading5": "Heading 5",
```

Comment:
I think @remidej has defined block items as "components.Blocks.**blocks**.heading3" , could you please pull latest changes from [feature/rich-text-editor](https://github.com/strapi/strapi/tree/feature/rich-text-editor)